lib/tasks/boilerplate.js

```diff
@@ -22,7 +22,8 @@ const buildTree = (root, name) => {
 		generate('/demos/src/demo.mustache', template.demoMustache(name)),
 		generate('/demos/src/pa11y.mustache', template.demoPa11y(name)),
 		generate('/demos/src/demo.scss', template.demoSass(name)),
-		generate(`/src/js/${name.original}.js`, template.srcJs(name)),
+		generate(`/src/js/${name.plainName}.js`, template.srcJs(name)),
+		generate('/src/scss/_brand.scss', ''),
```

Comment:
A common pattern for branded components is to wrap `oBrandGet` and `oBrandSupportsVariant` in a private, component specific mixin `_oComponentGet`, and`_oComponentSupports`. Maybe it's worth adding that to the boilerplate?

Here's some examples:
- https://github.com/Financial-Times/o-table/blob/master/src/scss/_brand.scss#L3
- https://github.com/Financial-Times/o-overlay/blob/master/src/scss/_brand.scss#L3
- https://github.com/Financial-Times/o-buttons/blob/master/scss/_brand.scss#L3

Reply:
@rowanmanning should we look at this tomorrow?

Reply:
Added! 😊